stable/vpa/templates/updater-deployment.yaml

```diff
@@ -18,7 +18,10 @@ spec:
       annotations:
         {{- toYaml . | nindent 8 }}
     {{- end }}
+    {{- with .Values.updater.podLabels }}
       labels:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
```

Comment:
```suggestion
      labels:
      {{- with .Values.updater.podLabels }}
        {{- toYaml . | nindent 8 }}
      {{- end }}
```

Reply:
This has been addressedSubmitted requested changes